lib/proxy/pluggable.rb

```diff
@@ -100,6 +86,7 @@ def default_settings(a_hash = {})
     end
 
     def initialize_after(*module_names)
+      logger.warn("#{plugin_name}: 'initialize_after' has been deprecated and will be removed in future versions.")
```

Comment:
That's all fine, thanks.  I just wanted to be clear about the new behaviour of the method, raising the exception's good.